packages/lib/src/index.ts

```diff
@@ -0,0 +1,107 @@
+import type { WindowAI } from "~contents/inpage"
```

Comment:
how does it know where this is?

Reply:
I think this can be fixed by using tsconfig path and a custom virtual namespace - since type sharing doesn't require ts transpile and just type compilation, the ts path should suffice

Reply:
Would it be better have an interface defined in this lib package (that includes all the window.ai methods), and have the extension’s WindowAI just implement the interface?